### ISSUE TITLE
[Meross] [Smart Plug] is not supported

### ISSUE THREAD

**Issue by rrounds** [donavanbecker/homebridge-meross#541]
Per the following log's request, opening this feedback:

> [10/29/2023, 5:12:17 PM] [Meross] [Smart Plug] is not supported, do create a github issue https://bit.ly/hb-meross-issues with info:
> {"uuid":"2004092609596990810148e1e9198b14","onlineStatus":2,"devName":"Smart Plug","devIconId":"device046_us","bindTime":1595471823,"deviceType":"mss120","subType":"us","channels":[{},{"type":"Switch","devName":"Switch 1","devIconId":"device001"},{"type":"Switch","devName":"Switch 2","devIconId":"device001"}],"region":"us","fmwareVersion":"4.1.4","hdwareVersion":"4.0.0","userDevIcon":"","iconType":1,"domain":"mqtt-us.meross.com","reservedDomain":"mqtt-us.meross.com","hardwareCapabilities":[],"firmware":"4.1.4","hbDeviceId":"2004092609596990810148e1e9198b14","model":"MSS120"}

Thank your for your great work!

**Comment by bwp91** [comment]
Hi @rrounds 

I have just published a beta version of the plugin which should get this device working :)

**Comment by rrounds** [comment]
Awesome - thank you! 

I’ll get that update installed tonight.

**Comment by rrounds** [comment]
Loaded the beta - It seems to be working and recognizing the device, but now a switch is not online. That's likely my end and I'll track it down. Let me know if you care to see the logs after restarting. many thanks!

**Comment by bwp91** [comment]
did this work in the end?

**Comment by rrounds** [comment]
Yes, I’m sorry I thought I added to the thread. Thank you!- Ray ROn Dec 29, 2023, at 9:07 AM, Ben ***@***.***> wrote:﻿
did this work in the end?
